### DISCUSSION TITLE
Logo is huge on firefox.

### DISCUSSION THREAD

**Discussion by niiccoo2** [wavelog/wavelog#1340] [Q&A] [answered]
Works fine in chrome, Inspect element gives these errors. 

This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. dashboard
Uncaught ReferenceError: quickswitcher_show_activebadge is not defined
    <anonymous> http://192.168.1.228:1080/index.php/dashboard:588

**Comment by HB9HIL** [comment]
What is that :588 at the end of your URL?

**Reply by niiccoo2** [reply]
That was part of the html error. I somehow got it working.. Try setting it up with chrome and rebooting after... It was weird

**Reply by HB9HIL** [reply]
Did you had Cloudlog open in the same Browser Window?

**Comment by niiccoo2** [comment]
No

**Reply by HB9HIL** [reply]
So the reason for that "glitch" was probably a caching issue

**Reply by niiccoo2** [reply]
Ok, One more question is there a way to change the number of QSO showed on the map in the log book and on the homescreen

**Reply by HB9HIL** [reply]
Not yet, sorry

**Reply by HB9HIL** [reply, marked as answer]
PS: to avoid such caching issues you can do `Ctrl + Shift + F5` on Windows or `Cmd + Shift + R` on Mac. This reloads the page in the webbrowser and ignores the cache
